.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
 
 sudo: false
-
+cache: pip
```

Comment:
This doesn't work with travis ``install`` section override: https://github.com/travis-ci/travis-ci/issues/3239